tailwind.config.js

```diff
@@ -2,7 +2,34 @@
 module.exports = {
     content: ["./pages/**/*.{js,ts,jsx,tsx}", "./src/**/*.{js,ts,jsx,tsx}"],
     darkMode: "media",
-    theme: {},
+    theme: {
```

Comment:
@dimitribarbot 

Thank you for this PR.
We like the idea of having `primary` and `secondary` colors for the component.

`theme` configs in the `tailwind.config.js` file can only be used during development while using the [Playground](https://github.com/onesine/react-tailwindcss-datepicker/blob/master/README.md#playground). 
Thus, adding custom tailwind config may not work in production.

Reply:
@JefteCaro Indeed, the problem with this PR is that we do not publish the tailwind.config.js file. Also, we don't generate a particular CSS file. In development on the "Next" server, this will probably work, while in production, the styles will not apply.

@dimitribarbot , can you make another PR that would work in production. That would be really great.

Reply:
@dimitribarbot 

Your PR includes configuration of additional color variant `primary` and `secondary`. Which generates new class names like `text-primary`, `text-secondary`, `bg-primary`, `bg-secondary`, etc.
Which, are not by default supported by `tailwindcss`.

By including the `primary` and `secondary` options to the `const COLORS` will give an impression that this should work from their end, but, the css class will not be generated.